pilot/pkg/bootstrap/sidecarinjector.go

```diff
@@ -37,6 +37,10 @@ import (
 var (
 	injectionWebhookConfigName = env.RegisterStringVar("INJECTION_WEBHOOK_CONFIG_NAME", "istio-sidecar-injector",
 		"Name of the mutatingwebhookconfiguration to patch, if istioctl is not used.")
+	tlsCertFile = env.RegisterStringVar("INJECTION_WEBHOOK_TLS_SERVER_CERT_CHAIN", filepath.Join(dnsCertDir, "cert-chain.pem"),
```

Comment:
Is this really just for injection? Are you/users going to have different certs for CA, injection, validation, XDS, etc?

Reply:
It's going to be a single deployment with the same certs and the same identity. 

For Pilot, the plaintext listener will be fronted with an envoy and its certs will be delivered via `ISTIO_META_TLS_*` env vars (after https://github.com/istio/istio/issues/21833#issuecomment-615169432). 

We currently don't have Galley enabled but with istiod `/validate` will also require the same certs.

Reply:
Working on some updates to make it generic to handle all cases.